app/assets/stylesheets/application.css

```diff
@@ -145,3 +145,15 @@
   font-style: italic;
   font-size: 10px;
 }
+.grey {
+  color:white;
```

Comment:
this one is quite useful. its the grey "update pet" style

Reply:
Nice